shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -173,6 +174,10 @@ @implementation FlutterViewController {
   // The project to run in this controller's engine.
   FlutterDartProject* _project;
 
+  // The plugin used to handle mouse cursor. This is not an FlutterPlugin, so must be owned
+  // separately.
```

Comment:
Per the other comment I don't think this is true, in which case you wouldn't need an ivar for it.